configs/components/curl.rb

```diff
@@ -4,7 +4,12 @@
   pkg.url "http://buildsources.delivery.puppetlabs.net/curl-#{pkg.get_version}.tar.gz"
 
   pkg.build_requires "openssl"
-  pkg.build_requires "puppet-ca-bundle"
+  pkg.build_requires "puppet-ca-bundle" unless platform.architecture == "s390x"
```

Comment:
Can you put a comment here about how this is currently a known issue and should be fixed?